docs/UserGuide.md

```diff
@@ -180,16 +180,24 @@ Examples:
 
 ### Adding a note to clients : `note`
 
-Format: `note INDEX nt/NOTE`
+Format: `note INDEX [NOTE] `
 
 * Edits the note of the client specified by `INDEX`. The index refers to the index number shown in the displayed client list. The index **must be a positive integer** 1, 2, 3, …​
-* Existing note will be updated to the input note.
+* Existing note will be updated to the note specified.
+* If no note is given, the note for the client at the specified index will be deleted.
 
 > While this can also be done using the `edit` command, this `note` command serves as a faster way for users to directly modify a note.
 
 Examples:
-*  `note 1 nt/History of asthma` - Edits the note of the 1st client to `History of asthma`.
-*  `note 2 nt/Previously sprained both ankles` - Edits the note of the 2nd client to `Previously sprained both ankles`.
+
+* `note 1 History of asthma` - Edits the note of the 1st client to `History of asthma`.
+* `note 2 Previously sprained both ankles` - Edits the note of the 2nd client to `Previously sprained both ankles`.
```

Comment:
Will using `changes` instead of `edits` here be clearer to the user?

```suggestion
* `note 1 History of asthma` - Changes the note of the 1st client to `History of asthma`.
* `note 2 Previously sprained both ankles` - Changes the note of the 2nd client to `Previously sprained both ankles`.
```

Reply:
Sure